llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -18835,6 +18835,8 @@ static SDValue tryFoldSelectIntoOp(SDNode *N, SelectionDAG &DAG,
   case ISD::ADD:
   case ISD::OR:
   case ISD::XOR:
+  case ISD::UMIN:
+  case ISD::UMAX:
```

Comment:
I think SMAX/SMIN would also be good to add, and yield similar results?

Reply:
This is mentioned in the description "smin/smax are a bit harder, and will be a separate change."